tap_google_sheets/schema.py

```diff
@@ -96,10 +96,13 @@ def get_sheet_schema_columns(sheet):
 
             col_val = None
             if column_effective_value == {}:
-                column_effective_value_type = 'stringValue'
-                LOGGER.info('WARNING: NO VALUE IN 2ND ROW FOR HEADER. SHEET: {}, COL: {}, CELL: {}2.'.format(
-                    sheet_title, column_name, column_letter))
-                LOGGER.info('   Setting column datatype to STRING')
+                if ("numberFormat" in first_value.get('effectiveFormat', {})):
+                    column_effective_value_type = "numberValue"
+                else:
+                    column_effective_value_type = 'stringValue'
+                    LOGGER.info('WARNING: NO VALUE IN 2ND ROW FOR HEADER. SHEET: {}, COL: {}, CELL: {}2.'.format(
+                        sheet_title, column_name, column_letter))
+                    LOGGER.info('   Setting column datatype to STRING')
```

Comment:
Any reason for leaving initial spaces?

Reply:
It was done to keep the loggers consistent